### PR TITLE
Add a new member request template

### PR DESCRIPTION
**Pull request by linsun**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/organization-membership-request.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - Contributing to working group or community discussions
+- [ ] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines.
+- [ ] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application
+
+### Sponsors (Please mention sponsors’ github ids here)
+- (at)sponsor-1
+- (at)sponsor-2
+
+### List of contributions to the Istio project
+- PRs reviewed / authored
```
Comment (sdake) [inline .github/ISSUE_TEMPLATE/organization-membership-request.md]: reviewing and authoring are two distinct activities - would recommend them on separate lines.  Also as mentioned on the TOC call by @andraxylia there are queries that can be used to gather this information.

File: .github/ISSUE_TEMPLATE/organization-membership-request.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
+name: Organization Membership Request
+about: Request membership in an Istio Org
+
+---
+
+### GitHub Username
+e.g. (at)example_user
+
+### Organization you are requesting membership in
```
Comment (geeknoid) [inline .github/ISSUE_TEMPLATE/organization-membership-request.md]: I don't think we need this, there's only one org.

File: .github/ISSUE_TEMPLATE/organization-membership-request.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+---
+name: Organization Membership Request
+about: Request membership in an Istio Org
+
+---
+
+### GitHub Username
```
Comment (geeknoid) [inline .github/ISSUE_TEMPLATE/organization-membership-request.md]: Isn't this implicit based on who opens the issue?

File: .github/ISSUE_TEMPLATE/organization-membership-request.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+e.g. (at)example_user
+
+### Organization you are requesting membership in
+e.g. (at)istio
+
+### Company Affiliation or Individual
+e.g. Google, IBM, VMware, Red Hat or XX etc, otherwise, Individual
+
+### Requirements
+- Subscribed to istio-dev@googlegroups.com (istio-dev@googlegroups.com).
```
Comment (geeknoid) [inline .github/ISSUE_TEMPLATE/organization-membership-request.md]: Make this a link:

[istio-dev@](https://groups.google.com/forum/#!forum/istio-dev)

File: .github/ISSUE_TEMPLATE/organization-membership-request.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+- Actively contributing to 1 or more areas.
+- Sponsored by 2 members.
+
+- [ ] I have reviewed the community membership guidelines (https://github.com/istio/community/blob/master/ROLES.md#member & https://github.com/istio/community/blob/master/CONTRIBUTING.md)
```
Comment (geeknoid) [inline .github/ISSUE_TEMPLATE/organization-membership-request.md]: Can these two be made markdown links?

File: .github/ISSUE_TEMPLATE/organization-membership-request.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+- [ ] I have enabled 2FA on my GitHub account (https://github.com/settings/security)
+- [ ] I have subscribed to the istio-dev@googlegroups.com email list
```
Comment (geeknoid) [inline .github/ISSUE_TEMPLATE/organization-membership-request.md]: Make this a a link: https://groups.google.com/forum/#!forum/istio-dev

File: .github/ISSUE_TEMPLATE/organization-membership-request.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+- [ ] I have made multiple contributions to the project or community. Contributions may include, but are not limited to:
+    - Authoring or reviewing PRs on GitHub
+    - Filing or commenting on issues on GitHub
+    - Contributing to working group or community discussions
+- [ ] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines.
+- [ ] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application
+
+### Sponsors (Please mention sponsors’ github ids here)
```
Comment (geeknoid) [inline .github/ISSUE_TEMPLATE/organization-membership-request.md]: Just:

### Sponsor GitHub Ids

File: .github/ISSUE_TEMPLATE/organization-membership-request.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+- (at)sponsor-1
+- (at)sponsor-2
+
+### List of contributions to the Istio project
+- PRs reviewed 
+- PRs authored
+- Issues responded to
+- Workgroup(s) I am involved with
+
+### Slack access:
```
Comment (geeknoid) [inline .github/ISSUE_TEMPLATE/organization-membership-request.md]: Remove colon

File: .github/ISSUE_TEMPLATE/organization-membership-request.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+- What is your email address associated with Slack?
```
Comment (geeknoid) [inline .github/ISSUE_TEMPLATE/organization-membership-request.md]: Please change to:

If you'd like to join our developer Slack workspace, what is the email address you have associated with Slack?

File: .github/ISSUE_TEMPLATE/organization-membership-request.md [hunk 10]
```diff
@@ -0,0 +1,42 @@
+---
```
Comment (geeknoid) [inline .github/ISSUE_TEMPLATE/organization-membership-request.md]: We should add a note about this process in one or two places in the other files in the community repo so folks can discover this.

Comment (linsun) [reply .github/ISSUE_TEMPLATE/organization-membership-request.md]: yes, definitely.  waiting on for this to be published, then we can link to it.

File: .github/ISSUE_TEMPLATE/organization-membership-request.md (hunk 3 above)
Comment (linsun) [reply .github/ISSUE_TEMPLATE/organization-membership-request.md]: it is, if it is opened by the same person, but we don't require to be the same person.

File: .github/ISSUE_TEMPLATE/organization-membership-request.md [hunk 11]
```diff
@@ -0,0 +1,39 @@
...
+name: Organization Membership Request
+about: Request membership in an Istio Org
```
Comment (geeknoid) [inline .github/ISSUE_TEMPLATE/organization-membership-request.md]: an -> the
Org -> organization

File: .github/ISSUE_TEMPLATE/organization-membership-request.md [hunk 12]
```diff
@@ -0,0 +1,39 @@
...
+
+### Requirements
+- Subscribed to istio-dev@googlegroups.com (https://groups.google.com/forum/#!forum/istio-dev).
+- Actively contributing to 1 or more areas.
+- Sponsored by 2 members.
+
+- [ ] I have reviewed the community membership guidelines (https://github.com/istio/community/blob/master/ROLES.md#member & https://github.com/istio/community/blob/master/CONTRIBUTING.md)
+- [ ] I have enabled 2FA on my GitHub account (https://github.com/settings/security)
+- [ ] I have subscribed to the istio-dev@googlegroups.com (https://groups.google.com/forum/#!forum/istio-dev) email list
+- [ ] I have made multiple contributions to the project or community. Contributions may include, but are not limited to:
```
Comment (cmluciano) [inline .github/ISSUE_TEMPLATE/organization-membership-request.md]: Would it be inappropriate to attach a number of contributions?

Should we also state a general breadth in the commits (i.e. more than just 10 dependency bumps)?

Comment (sdake) [reply .github/ISSUE_TEMPLATE/organization-membership-request.md]: @cmluciano imo its more about impact than raw stats.  For example, a new contributor submits a PR to the docs repo that enables delivery of helm charts via the website.  Huge impact, 1 commit.  That individual imo would be a prime candidate for membership once more work is done.  Whereas people fixing typos all over the code base with 30 commits - valued improvements but not necessarily impactful.

Can we can leave it up to the sponsors to decide?

Cheers
-steve

**Comment by andraxylia** [comment]
A simple query:
https://github.com/istio/istio/commits/master?author=<userid>

PRs by date:
https://github.com/istio/istio/pulls?utf8=%E2%9C%93&q=is%3Apr+author%3A<USERid>+created%3A%3E<YYYY-MM-DD>

**Comment by geeknoid** [comment]
Maybe we should put these in the template itself to help applicants?

**Comment by linsun** [comment]
@andraxylia I can't get the 2nd query working.  feel free to submit a PR to the template to include the help links.

**Comment by andraxylia** [comment]
To get the equivalent, go to pull requests, delete is:open and add to the query created:>YYYY-MM-DD.

**Pull request merged**